includes/class-freemius.php

```diff
@@ -264,9 +269,24 @@ private function __construct( $slug ) {
 
 			$this->_plugin = FS_Plugin_Manager::instance( $this->_slug )->get();
 
+			/**
+			 * @todo Check if there is a better way to set $this->_storage->was_plugin_loaded to true for "theme" module.
+			 * If there is a better way to do that, the following if-else block will not be needed anymore and the original
+			 * code can be brought back.
+			 *
+			 * Since the current code immediately sets $this->_storage->was_plugin_loaded to "true" for "theme" module,
+			 * should_stop_execution() will return "false" and the "few-plugin-tweaks" admin notice can be added with an
+			 * empty title. So the following if-else block ensures that the admin notice will always have a title.
+			 */
+			if ( ! is_object( $this->_plugin ) && $this->is_theme() ) {
```

Comment:
@vovafeldman Because `get_plugin_name()` will do more work if `$this->_plugin_name` is not yet set. So if `$this->_plugin` is already set and `$this->_plugin_name` is not yet set, that "more work" can be avoided if I will just use `$this->_plugin`. I may need to dig deeper to understand why this behavior only happens with plugins (`$this->_plugin` already set and `$this->_plugin_name` not yet set).